test/dummy/app/models/orange.rb

```diff
@@ -0,0 +1,9 @@
+class Orange < ActiveRecord::Base
+  has_paper_trail class_name: "OrangeVersion"
+  default_scope -> { where(:name => 'orange') }
```

Comment:
this default scope is weird but needed something that would result in Version#item not returning the record